vp9/draft-ietf-payload-vp9-06.xml

```diff
@@ -957,23 +958,27 @@ N_G: | TID |U| R |-|-| (OPTIONAL)                 .
               <t>The clock rate in the "a=rtpmap" line MUST be 90000.</t>
 
               <t>The parameters "max-fs", and "max-fr", MUST be included in
-              the "a=fmtp" line of SDP if SDP is used to declare receiver capabilities.
-				These parameters are expressed as a
-              media subtype string, in the form of a semicolon separated list
-              of parameter=value pairs.</t>
+              the "a=fmtp" line of SDP if SDP is used to declare receiver
```

Comment:
I think we want something similar to S 8.1 of RFC 6184 that outlines the possible profile id values and what profile names they correspond to.